package.json

```diff
@@ -37,7 +37,7 @@
     "css-element-queries": "^0.4.0",
     "d3": "^3.5.16",
     "intl": "^1.2.4",
-    "intl-messageformat": "^7.2.1",
+    "intl-messageformat": "^9.9.4",
```

Comment:
Technically `^9.9.4` is not the same as `9.9.4` used in ovirt-web-ui.  When you added it, yarn picked up `9.11.4`.  That's why the offline build fails.